### PR TITLE
Elasticsearch rails ha update

### PR DESCRIPTION
**Pull request by pkarman**

### CODE REVIEW

File: lib/tasks/search.rake [hunk 1]
```diff
@@ -0,0 +1,24 @@
+require 'elasticsearch/rails/ha/tasks'
+
+namespace :search do
+  desc 're-index all records, in parallel'
+  task index: :environment do
```
Comment (jessieay) [inline lib/tasks/search.rake]: should we be using this task elsewhere in the codebase?

**Comment by jessieay** [comment]
code climate is mad at you :fire

File: lib/tasks/search.rake (hunk 1 above)
Comment (pkarman) [inline lib/tasks/search.rake]: It's possible that the /admin button to re-index should use the stage+promote tasks in order to provide zero-downtime index rebuilds. But that can be an optimization later.

**Comment by pkarman** [comment]
codeclimate, cruel mistress, you are :green_heart: again!

cc/ @jessieay

File: spec/lib/roles_conversion_spec.rb [hunk 2]
```diff
@@ -1,8 +0,0 @@
-describe RolesConversion do
```
Comment (jessieay) [inline spec/lib/roles_conversion_spec.rb]: did you mean to delete this?

Comment (pkarman) [inline spec/lib/roles_conversion_spec.rb]: yes. see https://18f.slack.com/archives/cap/p1454714455001197

**Comment by rememberlenny** [comment]
Merged

**Pull request closed without merging**